contracts/game-contract/GameContract.sol

```diff
@@ -28,6 +34,9 @@ contract GameContract is Ownable {
     // Remove list Users
     event SignerListRemove(uint256 indexed totalAddedUser, uint256 indexed timestamp);
 
+    // Init event
+    event Initialize(address indexed owner, uint256 indexed timestamp);
```

Comment:
Should emit following information:
- Contract address
- Owner addresss
- Timestamp